provisioner/components/k8s/setup_playbook/final_setup/.travis.yml

```diff
@@ -0,0 +1,29 @@
+---
+language: python
+python: "2.7"
```

Comment:
are we using python 2.7 or 3.6?

Reply:
We are using python 3.6 only but it's a generated file that gets generated when we create an ansible role

Reply:
Updated with python 3.6